Sources/Datadog/URLSessionAutoInstrumentation/Interception/URLFiltering/InternalURLsFilter.swift

```diff
@@ -10,8 +10,8 @@ import Foundation
 internal struct InternalURLsFilter {
     private let internalURLPrefixes: Set<String>
 
-    init(configuration: FeaturesConfiguration.URLSessionAutoInstrumentation) {
-        self.internalURLPrefixes = configuration.sdkInternalURLs
+    init(internalURLs: Set<String>) {
```

Comment:
just like your change in `FirstPartyURLFilter`, we can do `init(urls: Set<String>)` here maybe

Reply:
Less verbose and still on point, I'll change that!